spring-xsuaa/src/main/java/com/sap/cloud/security/xsuaa/XsuaaServicesParser.java

```diff
@@ -139,9 +139,9 @@ private static JSONObject getJSONObjectFromTag(final JSONArray jsonArray, String
 			JSONObject binding = (JSONObject) jsonArray.get(i);
 			JSONArray tags = (JSONArray) binding.get(TAGS);
 
+			Optional<String> planName = Optional.ofNullable(binding.getAsString("plan"));
+			boolean isApiAccessPlan = (planName.isPresent() && planName.get().equals("apiaccess")) ? true : false;
```

Comment:
You could remove `? true : false` here and just use the result of the expression before the `?` operator :)